cmd/spire-server/cli/run/run.go

```diff
@@ -95,6 +95,9 @@ type serverConfig struct {
 	ProfilingFreq    int      `hcl:"profiling_freq"`
 	ProfilingNames   []string `hcl:"profiling_names"`
 
+	// Temporary configurables
+	UseLegacyDownstreamX509CATTL *bool `hcl:"use_legacy_downstream_x509_ca_ttl"`
```

Comment:
since it is temporary, can you add a deprecated comment so we can remove this in the future? 
I'm not sure when..

Reply:
Sure. This should be removed in 1.12.0. I will add a comment.